packages/network/src/logic/node/Node.ts

```diff
@@ -408,4 +472,8 @@ export class Node extends EventEmitter {
     emitJoinFailed(streamPartId: StreamPartID, error: string): void {
         this.emit(streamPartId, error)
     }
+
+    isProxiedSubscription(streamPartId: StreamPartID): boolean {
+        return this.proxyStreamConnectionManager.isSubscribeOnlyStream(streamPartId)
```

Comment:
Should this return true also for publisher-only streams?

Reply:
No, it is exclusively used for proxy subscriptions